portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,67 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//Training of fecth()
```

Comment:
Could you make the comment clearer? It's not clear to me how "training of fetch" relates to "addGreetingElmer"